nifi-docs/src/main/asciidoc/record-path-guide.adoc

```diff
@@ -1122,6 +1122,35 @@ Would yield a value of `2`. We could also use this as a filter, such as:
 Which will return the `id` element with a value of `1234`.
 
 
+=== map
+
+Creates a map of Strings with the given parameters. For example, if we have the following record:
+
+----
+{
+    "firstName": "John",
+    "lastName": "Snow"
+}
+----
+
+We could use the `UpdateRecord` processor with
+
+----
+/fullName => map("firstName", /firstName, "lastName", /lastName)
```

Comment:
Would it be better to qualify the function name to something else? The name `map` can also connote other types of streaming functions in the context of Java, so maybe something more specific would be better.

Reply:
Perhaps `mapOf` analogous to the newer `Map.of()` Java utility method?